skrifa/src/outline/autohint/latin/edges.rs

```diff
@@ -223,16 +277,25 @@ pub(crate) fn compute_blue_edges(
     scale: &Scale,
     unscaled_blues: &[UnscaledBlue],
     blues: &[ScaledBlue],
+    group: ScriptGroup,
 ) {
-    if axis.dim != Axis::VERTICAL {
+    // For the default script group, we only handle vertical blues
```

Comment:
because ...

Reply:
... FreeType does this. I added a link to the relevant code.

In reality, FT doesn't use any alignment zones in the horizontal direction but the infra exists for it. I believe this is an attempt at better hinting for vertical text but the code is permanently disabled with an `#undef`. After this all lands, I think it's worth considering simplifying our code and removing this altogether. Filed #1140